StandardLibrary/runtimes/go/ImplementationFromDafny-go/FileIO/externs.go

```diff
@@ -25,11 +36,20 @@ func (_static CompanionStruct_Default___) INTERNAL_ReadBytesFromFile(path _dafny
 }
 
 func (_static CompanionStruct_Default___) INTERNAL_WriteBytesToFile(path _dafny.Sequence, bytes _dafny.Sequence) (isError bool, errorMsg _dafny.Sequence) {
-	p := _dafny.SequenceVerbatimString(path, false)
+	p := func() string {
```

Comment:
I don't quite understand this change -- maybe Shubham should check this

Reply:
NVM -- old code was buggy and untested until https://github.com/aws/aws-cryptographic-material-providers-library/pull/1071

Reply:
Stolen from Shubham in the first place.